conf/modules.config

```
@@ -257,7 +257,7 @@ process {
     }
 
     withName: MEGAHIT {
-        ext.args   = { params.megahit_options ? params.megahit_options + "-m ${task.memory.toBytes()}" : "-m ${task.memory.toBytes()}" }
+        ext.args   = { params.megahit_options ? params.megahit_options + " -m ${task.memory.toBytes()}" : " -m ${task.memory.toBytes()}" }
```

```suggestion
        ext.args   = { params.megahit_options ? params.megahit_options + " -m ${task.memory.toBytes()}" : "-m ${task.memory.toBytes()}" }
```
the second white space isnt required I think, but might not hurt either.
Whats the default of megahit_options? If it is `""`, one could also do 
`{ params.megahit_options + " -m ${task.memory.toBytes()}" }` ?

---

OK yeah, lets do that

---

There are no defaults, so it should be fine to inject `-m` as is

---

Internet terrible here, github interface not working, so did it manually in https://github.com/nf-core/mag/pull/710/commits/52ccb1cdf4e4d4861a4521b51da3c6f6acec84ba